scripts/helpers.sh

```diff
@@ -55,6 +55,19 @@ load_status() {
   fi
 }
 
+temp_status() {
+  local temp=$1
+  cpu_medium_thresh=$(get_tmux_option "@cpu_temp_medium_thresh" "80")
+  cpu_high_thresh=$(get_tmux_option "@cpu_temp_high_thresh" "90")
```

Comment:
You're using cpu thresh also for gpu temp
https://github.com/tmux-plugins/tmux-cpu/pull/61/files#diff-ed83f3a8b96dd29f629702bd1f6bba0e0d7796be7288011a9172b718578151f4R25

Reply:
good spot